src/rust/grapl-observe/src/metric_reporter.rs

```diff
@@ -89,7 +89,7 @@ where
             tags,
         )?;
         let time = self.format_time_for_cloudwatch((self.utc_now)());
-        write!(
+        writeln!(
             self.out.as_mut(),
             "MONITORING|{}|{}|{}\n",
```

Comment:
writeln already adds the \n, so anywhere you have a writelin! remove the \n in the string

Reply:
Resolved